services/webnn/ort/graph_impl_ort.cc

```diff
@@ -103,12 +103,8 @@ GraphImplOrt::CreateAndBuildOnBackgroundThread(
 
   const OrtApi* ort_api = GetOrtApi();
   ScopedOrtSessionOptionsPtr session_options;
-  OrtStatus* status =
-      ort_api->CreateSessionOptions(session_options.GetAddressOf());
-  if (status) {
-    std::string_view msg = ort_api->GetErrorMessage(status);
-    LOG(ERROR) << "[WebNN] Failed to create session options: " << msg;
-    ort_api->ReleaseStatus(status);
+  if (GET_STATUS_PTR(
```

Comment:
Would it be more clear by defining a `FAILED(ort_call)` or `ORT_CALL_FAILED(ort_call)`?